src/main/java/duke/app/Duke.java

```diff
@@ -0,0 +1,165 @@
+package duke.app;
+
+import duke.file.FileOperation;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.ArrayList;
+import  java.util.Scanner;
+
+public class Duke {
+
+    static final int MAX_NO_OF_TASKS = 100;
+
+    public static void main(String[] args) {
+        showWelcomeMessage();
+
+        ArrayList<Task> userTasks = new ArrayList<>(MAX_NO_OF_TASKS);
+        boolean exitLoopStatus = false;
+        Scanner inputCommand = new Scanner(System.in);
+        String userInput;
+        FileOperation.readFile(userTasks);
+
+        while(true){
+            userInput = inputCommand.nextLine();
+            String[] individualWords = userInput.split(" ", 2);
+            switch(individualWords[0].toLowerCase()){
+            case "list":
+                displayListOfActivities(userTasks);
+                break;
+            case "todo":
+                createTodoTask(userTasks, individualWords);
+                FileOperation.writeFile(userTasks);
+                break;
+            case "deadline":
+                createDeadlineTask(userTasks, individualWords);
+                FileOperation.writeFile(userTasks);
+                break;
+            case "event":
+                createEventTask(userTasks, individualWords);
+                FileOperation.writeFile(userTasks);
+                break;
+            case "done":
+                markActivityAsDone(userTasks, individualWords);
```

Comment:
Same issue: if you choose to use the word task, maybe this method should be called "markTaskAsDone".